source/app/utils/document.js

```diff
@@ -565,3 +561,5 @@ export function getPageWordsBySearch(document, pageNumber, searchQuery) {
   }, [])
   return matchingWords
 }
+
+export const getAreRedactionsOnDocument = document => document.redactions && Object.values(document.redactions).some((redactionsOnPage) => Object.values(redactionsOnPage).length > 0);
```

Comment:
What is purpose of the first check here?
If there is no redactions key, wouldnt document.redactions be undefined?

Reply:
Fixed